Task/t1.py

```diff
@@ -0,0 +1,25 @@
+import os
+from Utils.read_files import read_files
+
+
+file_values = read_files()
+
+Temperatures = {}
```

Comment:
Variables must always be small case!! (unless they are constants)

Reply:
ok